projects/plugins/boost/app/modules/optimizations/cloud-css/class-cloud-css.php

```diff
@@ -199,18 +198,6 @@ public function handle_save_post( $post_id, $post ) { // phpcs:ignore VariableAn
 		}
 	}
 
-	/**
-	 * Handle when a critical CSS environment change is detected.
-	 */
-	public function handle_environment_change( $is_major_change, $change_type ) {
```

Comment:
I noticed this was added in https://github.com/Automattic/jetpack/pull/41516/files but was unused even there.